ml-lifecycle/feature_store/README.md

```diff
@@ -1,4 +1,68 @@
+
+
+# Prereqs
+
+The following policies need to be attached to the SageMaker execution role that you use to run this notebook:
+
+- AmazonSageMakerFullAccess
+- AmazonSageMakerFeatureStoreAccess
+- AmazonS3FullAccess
+
+Note that the AmazonS3FullAccess policy is not attached to your role by default if you choose to create a new role when you start your SageMaker Studio instance. If you don't see the required policies above are listed under Policy name, you can go to the IAM console, find your role, choose Attach Policies under Permissions, find the policies you are missing from the list, then choose Attach policy. For more information, see the [Developer Guide: SageMaker Roles](https://docs.aws.amazon.com/sagemaker/latest/dg/sagemaker-roles.html)
+
+
+# Data
+
+This notebook uses the publically available [Heart failure clinical records Data Set](https://archive.ics.uci.edu/ml/datasets/Heart+failure+clinical+records) that can be downloaded from the UCI machine learning Repository, as described in the notebook. The data set contains medical record information for a small sample of heart failure patients, including demographic, diagnostic and laboratory test data.
+
+**heart_failure_clinical_records_dataset.csv**  
+
+The dataset contains one table with thirteen (13) columns:
+
+- age: age of the patient (years)
+- anaemia: decrease of red blood cells or hemoglobin (boolean)
+- high blood pressure: if the patient has hypertension (boolean)
+- creatinine phosphokinase (CPK): level of the CPK enzyme in the blood (mcg/L)
+- diabetes: if the patient has diabetes (boolean)
+- ejection fraction: percentage of blood leaving the heart at each contraction (percentage)
+- platelets: platelets in the blood (kiloplatelets/mL)
+- sex: woman or man (binary)
+- serum creatinine: level of serum creatinine in the blood (mg/dL)
+- serum sodium: level of serum sodium in the blood (mEq/L)
+- smoking: if the patient smokes or not (boolean)
+- time: follow-up period (days)
+- (target)death event: if the patient deceased during the follow-up period (boolean)
+
+
+ # Approach
+ 
+![architecture diagram](/fs1.PNG)
+ 
+First, we'll prepare the data for feature store, create a Feature Group and then ingest our data in to the Feature Group. Our features will be available in the offline feature store within minutes. We then use the feature store to build a training dataset, fit a simple model and return predictions.
+
+# Clean Up
+
+In order to prevent ongoing charges to your AWS account, clean up any resources we spun up during this tutorial. An easy way to track and delete related training resources is to [tag them](https://docs.aws.amazon.com/general/latest/gr/aws_tagging.html) or create the resources all under the same [resource group](https://us-east-2.console.aws.amazon.com/resource-groups/home) so that later you can easily search and delete unwanted resources.
```

Comment:
If this notebook does not tag the resources for the user, I think it doesn't make sense to mention tagging, because they either have to add all the tagging themselves or they already ran the notebook and created all the resources without tagging them. It looks like the notebook cleanup section deletes the resources, so that's sufficient without tagging.